mteb/tasks/Audio/AudioZeroshotClassification/eng/SpeechCommands.py

```diff
@@ -0,0 +1,87 @@
+from __future__ import annotations
+
+from mteb.abstasks.Audio.AbsTaskAudioZeroshotClassification import (
+    AbsTaskAudioZeroshotClassification,
+)
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class ESC50ZeroshotClassification(AbsTaskAudioZeroshotClassification):
+    metadata = TaskMetadata(
+        name="SpeechCommandsZeroshot",
+        description="Sound Classification/Keyword Spotting Dataset. This is a set of one-second .wav audio files, each containing a single spoken English word or background noise. These words are from a small set of commands, and are spoken by a variety of different speakers. With total of 30 labels",
+        reference="https://huggingface.co/datasets/google/speech_commands",
+        dataset={
+            "path": "google/speech_commands",
+            "name": "v0.01",
+            "revision": "57ba463ab37e1e7845e0626539a6f6d0fcfbe64a",
+        },
+        type="AudioZeroshotClassification",
+        category="a2a",
+        eval_splits=["train"],
+        eval_langs=["eng-Latn"],
+        main_score="accuracy",
+        date=("2018-07-07", "2018-07-13"),
+        domains=[
+            "Spoken"
+        ],  # Replace with appropriate domain from allowed list?? No appropriate domain name is available
+        task_subtypes=["Keyword Spotting"],
+        license="cc-by-nc-sa-3.0",  # Replace with appropriate license from allowed list
```

Comment:
```suggestion
        license="cc-by-4.0",
```
Assumed from HF license?

Reply:
Fixed